README.md

```diff
@@ -339,31 +343,6 @@ ews.run(ewsFunction, ewsArgs, ewsSoapHeader)
 
 ```
 
-#### Use Encrypted Credentials for NTLM:
-This allows you to persist their password as separate hashes instead of as plain text.
-This utilizes the [options](https://github.com/SamDecrock/node-http-ntlm#options) available to the underlying NTLM lib.
-[Here](https://github.com/SamDecrock/node-http-ntlm#pre-encrypt-the-password) is an example from its README.
-Below is an example for this lib:
-```js
-const NTLMAuth = require('httpntlm').ntlm;
-const passwordPlainText = 'mypassword';
-
-// store the ntHashedPassword and lmHashedPassword to reuse later for reconnecting
-const ntHashedPassword = NTLMAuth.create_NT_hashed_password(passwordPlainText);
-const lmHashedPassword = NTLMAuth.create_LM_hashed_password(passwordPlainText);
-
-// exchange server connection info
-const ewsConfig = {
-    username: 'myuser@domain.com',
-    nt_password: ntHashedPassword,
-    lm_password: lmHashedPassword,
```

Comment:
If this is no longer supported, then this is a breaking change and you'll need to bump the version to `4.0.0`